llvm/include/llvm/Support/Base64.h

```diff
@@ -56,6 +56,32 @@ template <class InputBytes> std::string encodeBase64(InputBytes const &Bytes) {
 
 llvm::Error decodeBase64(llvm::StringRef Input, std::vector<char> &Output);
 
+// General-purpose Base64 encoder/decoder class wrapper.
+class Base64 {
+public:
+  using byte = uint8_t;
```

Comment:
Can we use `std::byte`?

Reply:
Thanks for the pointer. Since we have moved to C++17 and we intend only memory accesses here, std::byte is a good match. Will add the change in upcoming patch.

Reply:
Added in 526633fe54188134d77e033c512fafbab11ff4ba

Thanks